### PR TITLE
feat: #8292 (QBadge) add round prop

### PR DESCRIPTION
**Pull request by suyundukov**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Documentation
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [x] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar/tree/dev/docs) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by pdanpdan** [comment]
#8292

File: ui/src/components/badge/QBadge.sass [hunk 1]
```diff
@@ -27,3 +27,6 @@
   &--outline
     background-color: transparent
     border: 1px solid currentColor
+
+  &--round
+    border-radius: 50px
```
Comment (pdanpdan) [inline ui/src/components/badge/QBadge.sass:32]: Please change this to `1em` - I played a little and it looks like it's good

File: ui/src/components/badge/QBadge.styl [hunk 2]
```diff
@@ -27,3 +27,6 @@
   &--outline
     background-color: transparent
     border: 1px solid currentColor
+
+  &--round
+    border-radius: 50px
```
Comment (pdanpdan) [inline ui/src/components/badge/QBadge.styl:32]: Please change this to `1em` - I played a little and it looks like it's good

**Review by pdanpdan — CHANGES_REQUESTED** [review]

**Review by pdanpdan — APPROVED** [review]

**Comment by rstoenescu** [comment]
Thanks for contributing!
Changed the "round" to "rounded" to match the syntax with other components as QBadge with this prop is not perfectly circle.

**Comment by rstoenescu** [comment]
Nice work. A well done PR. Contains docs, examples, API.
I especially like the "1em" change which sealed the deal to merge it :)
Will be available in Quasar v1.15.2.

**Pull request merged**